app/views/registrations/login.html.erb

```diff
@@ -7,14 +7,18 @@
     <%= form.text_field :email, class:"form-control"%>
     <%= form.label :email, class:"form-label" %>
   </div>
-  <div class="mb-3">
+  <div class="">
     <%= form.password_field :password, id: 'user_password', class:"form-control"%>
     <%= form.label :password, class:"form-label" %>
   </div>
-  <div class="d-flex flex-row align-items-center mb-5">
+  <div class="d-flex flex-row align-items-center mb-2">
     <%= form.check_box :show_password, onchange: "showPassword()", class: "form-check-input me-2" %>
     <%= form.label :show_password, 'Show password', class: 'form-check-label' %>
   </div>
+  <div class="d-flex flex-row align-items-center mb-4">
```

Comment:
try to show remember_me check box just above login button.

Reply:
Done